lib/puppet/pops/validation/checker4_0.rb

```diff
@@ -649,13 +684,13 @@ def check_ResourceBody(o)
   end
 
   def check_ResourceDefaultsExpression(o)
-    if o.form && o.form != :regular
+    if o.form != 'regular'
```

Comment:
This is different from the original where 'form' was optional - is it now always a value ?